tests/aws/services/apigateway/test_apigateway_lambda.py

```diff
@@ -300,8 +288,15 @@ def test_lambda_aws_proxy_integration_non_post_method(
         uri=f"arn:aws:apigateway:{aws_client.apigateway.meta.region_name}:lambda:path/2015-03-31/functions/{lambda_arn}/invocations",
         credentials=role_arn,
     )
-    aws_client.apigateway.create_deployment(restApiId=api_id, stageName=stage_name)
 
+    # TODO: comment
+    aws_client.apigateway.put_gateway_response(
+        restApiId=api_id,
+        responseType="INTEGRATION_FAILURE",
+        responseParameters={},
+    )
+
```

Comment:
That seems like a left over from our session? Do you want to test some gateway responses here?